products/workers/src/content/platform/limits.md

```diff
@@ -312,6 +300,6 @@ Durable Objects are currently in beta. See [their docs](/learning/using-durable-
 
 - Storage values of up to 32 KiB (32768 bytes)
 
-- 500ms of CPU time per request, including websocket messages (this will be raised to match the Unbound limit after the beta period)
+- 30s of CPU time per request, including websocket messages
 
-Durable Objects have been built such that the number of objects in the system do not need to be limited. You can create and run as many separate objects as you want. The main limit to your usage of Durable Objects is the total storage limit per account, which we expect to raise after the beta period ends. If you need more storage in the meantime, please contact us.
+Durable Objects have been built such that the number of objects in the system do not need to be limited. You can create and run as many separate objects as you want. The main limit to your usage of Durable Objects is the total storage limit per account - if you need more storage, please contact us.
```

Comment:
It might also be worth mentioning here the practical limit on how many requests per second any individual object can handle?